database/migrations/2022_09_27_024653_update_users_table.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->dropColumn('name');
+            $table->string('first_name');
+            $table->string('last_name');
+            $table->string('username');
+            $table->boolean('is_admin')->default(false);
+            $table->boolean('is_active')->default(false);
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->string('name');
+            $table->dropColumn('first_name');
+            $table->dropColumn('last_name');
+            $table->dropColumn('username');
+            $table->dropColumn('is_admin');
+            $table->dropColumn('is_active');
```

Comment:
Recommend: Nên kiểm tra cột đó có tồn tại không trước khi drop
https://stackoverflow.com/questions/54176290/check-if-a-column-exists-in-laravel-migration-file